### ISSUE TITLE
[RFE] use tools.mypy from pyproject.toml

### ISSUE THREAD

**Issue by t184256** [realpython/pytest-mypy#153]
Configuring through `mypy.ini` works for me, but it'd be great if mypy invoked through pytest-mypy could pick up configuration from `tools.mypy` section of `pyproject.toml` as described in https://mypy.readthedocs.io/en/stable/config_file.html.

**Comment by dmtucker** [comment]
This should already be supported (but you need to use `tool.mypy`, not `tools`).
I will add a test to ensure this support doesn't break.

**Comment by t184256** [comment]
Oh no. Sorry for bothering, it's indeed me not being attentive enough and using `tools.mypy` instead of `tool.mypy`.

**Comment by dmtucker** [comment]
No worries! It'll be good to have that test :+1:

**Comment by dmtucker** [comment]
@djwessel That isn't specific to config in `pyproject.toml` actually. See https://github.com/realpython/pytest-mypy/issues/123#issuecomment-903174805

tl;dr: That option isn't terribly compatible with this plugin (which aims to let Pytest decide which files get checked).
